frontend/src/app/scratch/[slug]/opengraph-image.tsx

```diff
@@ -23,7 +24,8 @@ export default async function ScratchOG({ params }: { params: { slug: string }})
     const { scratch, parentScratch, compilation } = await getScratchDetails(params.slug)
 
     const compilers = await get("/compiler")
-    const preset = compilers["platforms"][scratch.platform].presets.find(p => p.id === scratch.preset)
+    const presets: api.Preset[] = compilers["platforms"][scratch.platform].presets
+    const preset = presets.find(p => p.id === scratch.preset)
```

Comment:
Ah - this has undone my recent change to grab the preset from the `/preset` endpoint

Reply:
oopsh! will fix